generators/import-jdl/index.js

```diff
@@ -70,10 +70,10 @@ class ImporterGenerator extends BaseGenerator {
             this.clientFramework = this.clientFramework || 'angularX';
             this.clientPackageManager = configuration.clientPackageManager;
             if (!this.clientPackageManager) {
```

Comment:
I'm gonna set NPM as default today and release a patch version for it.

Reply:
Thanks @MathieuAA : for what projects ?
Next week, I planned to do the same for jhipster-devbox, maybe I miss some other projects ?